www/front_src/src/Resources/Details/index.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+
+import { Paper, makeStyles, Divider } from '@material-ui/core';
+
+import { Status, Parent, Downtime, Acknowledgement } from '../models';
+import Header from './Header';
+import Body from './Body';
+import useGet from '../useGet';
+
+const useStyles = makeStyles(() => {
+  return {
+    details: {
+      height: '100%',
+      display: 'grid',
+      gridTemplate: 'auto auto 1fr / 1fr',
+    },
+    header: {
+      gridArea: '1 / 1 / 2 / 1',
```

Comment:
It could be useful to make a course about CSS grid

Reply:
@Thebarda Yes, we're going to do a PR in the end with all the translations. It's going to help us involving people in the review that are not frontend devs.